### PR TITLE
License

### PR DESCRIPTION
**Pull request by DOOduneye**
LICENSE

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,662 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (garrettladley) [inline LICENSE:633]:
```suggestion
    Student Activity Calendar, a better way to drive student engagement.
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+    Copyright (C) <year>  <name of author>
```
Comment (garrettladley) [inline LICENSE:634]:
```suggestion
    Copyright (C) 2024 David Oduneye and Garrett Ladley
```

File: LICENSE (hunk 1 above)
Comment (DOOduneye) [reply LICENSE:633]: mmmmmmmmm good catch

**Review by garrettladley — CHANGES_REQUESTED** [review]
See suggestions

**Pull request merged**